### PR TITLE
Adding Changes for 1.13 Release 

### PR DESCRIPTION
**Pull request by aditjind**
*Issue #, if available:* ODFE 1.13 Version Release #70

*Description of changes:*

* Verified that build is passing
* Verified that dashboards are coming up.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by ricardolstephen** [comment]
Could you add a gh issue to this pr

File: release-notes/opendistro-for-elasticsearch.perftop.release-notes-1.13.0.0.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+## 2020-11-19 Version 1.13.0.0 (Current)
```
Comment (ricardolstephen) [inline release-notes/opendistro-for-elasticsearch.perftop.release-notes-1.13.0.0.md:1]: Maybe change the date associated with this version?

Comment (aditjind) [reply release-notes/opendistro-for-elasticsearch.perftop.release-notes-1.13.0.0.md]: Done

**Comment by aditjind** [comment]
> Could you add a gh issue to this pr

Done

File: release-notes/opendistro-for-elasticsearch.perftop.release-notes-1.13.0.0.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+## 2021-02-01 Version 1.13.0.0 (Current)
+
+This is release of the Open Distro PerfTop, which is compatible with Open Distro Performance Analyzer 1.12
```
Comment (sruti1312) [inline release-notes/opendistro-for-elasticsearch.perftop.release-notes-1.13.0.0.md:3]: With PA 1.13*

File: release-notes/opendistro-for-elasticsearch.perftop.release-notes-1.13.0.0.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+Supported Elasticsearch version 7.10.2
+
+### Features
+
+### Bug fixes
+
+### Documentation
+
+### Maintenance
```
Comment (sruti1312) [inline release-notes/opendistro-for-elasticsearch.perftop.release-notes-1.13.0.0.md:12]: Can we add the name change commit under maintenance?

**Review by sruti1312 — APPROVED** [review]

**Review by ricardolstephen — APPROVED** [review]

**Pull request merged**